ui-v2/src/components/automations/automations-wizard/trigger-step/flow-multi-select.tsx

```diff
@@ -0,0 +1,133 @@
+import { useQuery } from "@tanstack/react-query";
+import { useDeferredValue, useMemo, useState } from "react";
+import { buildListFlowsQuery, type Flow } from "@/api/flows";
+import { Checkbox } from "@/components/ui/checkbox";
+import {
+	Combobox,
+	ComboboxCommandEmtpy,
+	ComboboxCommandGroup,
+	ComboboxCommandInput,
+	ComboboxCommandItem,
+	ComboboxCommandList,
+	ComboboxContent,
+	ComboboxTrigger,
+} from "@/components/ui/combobox";
+import { Typography } from "@/components/ui/typography";
+
+const MAX_FLOWS_DISPLAYED = 2;
```

Comment:
Can this be more dynamic? I'd like it to be based on the current size of the dropdown and the size of what needs to be displayed